yarn.lock

```diff
@@ -2,14 +2,15 @@
 # yarn lockfile v1
 
 
-"@aion-dk/js-client@1.1.3":
-  version "1.1.3"
-  resolved "https://registry.npmjs.org/@aion-dk/js-client/-/js-client-1.1.3.tgz"
-  integrity sha512-XGPZDLcHuFpu8lQllGph44G3aVUojBUBJdXDhjU1Ex4OHLFqEEmgsHtKOGoeQniaWwHpwgZSXnkLMch968/ENA==
+"@aion-dk/js-client@3.0.1":
+  version "3.0.1"
+  resolved "https://registry.yarnpkg.com/@aion-dk/js-client/-/js-client-3.0.1.tgz#00d7d092d8619fc5acc36730ce9d09646bc0c641"
+  integrity sha512-hga7lYCWo4Y3cYXaIHHcc2vo7fVptLZ/f427eV5C2TDgr9ZC9jyjVatOdI6zpKLzoPbQbJ7e3w8Oc7+LP/NHLw==
```

Comment:
It seems you used yarn, while it was previously built with npm. Is this intentional?

Reply:
This project has been using yarn for quite some time. Where do you see NPM being used here?

Reply:
From the file diff: 
![Screenshot 2023-04-28 at 09 07 48](https://user-images.githubusercontent.com/100700894/235078799-16d36c0c-0240-4957-91a2-063baddb15f1.png)

Reply:
I see. I have no idea why that's the case.